arduino-core/src/processing/app/helpers/FileUtils.java

```diff
@@ -85,7 +85,7 @@ public static void recursiveDelete(File file) {
         recursiveDelete(current);
       }
     }
-    deleteIfExists(file);
+    file.delete();
```

Comment:
It's explained in the second commit: `file is already checked for being not null, no need to check again`
the check is here: https://github.com/arduino/Arduino/pull/3141/files#diff-fb3046d74e0c4a32b94edd584638ecd3L76

BTW if you prefer to keep the defensive approach and check anyway, just drop the second commit and keep only the first one!